Terminal.Gui/ConsoleDrivers/CursesDriver/CursesDriver.cs

```diff
@@ -1083,7 +1107,7 @@ public void SetClipboardData (string text)
 			try {
 				BashRunner.Run ("xclip -selection clipboard -i", false, text);
 			} catch (Exception ex) {
-				throw new Exception (ex.Message);
+				throw new NotSupportedException ("Write to clipboard failed", ex);
```

Comment:
I think with the `ClipboardBase` abstract you don't need to have this try/catch block as there is already one in the base classes `SetClipboardData`

Reply:
Done. Thanks @tznind. If you see anything else not very well implemented, please let me know.